### PR TITLE
Use external webbrowser.Open helper.

### PR DESCRIPTION
**Pull request by frankban**
Also make the package compile without errors.

### CODE REVIEW

File: httpbakery/browser.go [hunk 1]
```diff
@@ -4,58 +4,22 @@ import (
...
-		cmd := exec.Command(args[0], args[1:]...)
-		cmd.Stdout = os.Stdout
-		cmd.Stderr = os.Stderr
-		cmd.Start()
-		go cmd.Wait()
-	} else {
+		return nil
+	}
+	if err == webbrowser.ErrNoBrowser {
 		fmt.Fprintf(os.Stderr, "Please open this URL in your browser to authorize:\n%s\n", url)
```
Comment (rogpeppe) [inline httpbakery/browser.go:22]: return nil?
I think in this case it's not useful to return an error.

**Comment by rogpeppe** [comment]
LGTM, thanks.
Please rebase before merging.

File: httpbakery/browser.go (hunk 1 above)
Comment (frankban) [inline httpbakery/browser.go:22]: Yeah, definitely forgot to add the "return nil" here.

**Comment by mhilton** [comment]
:+1:

**Pull request merged**